.bit/responses/5.1-Week 5 Step 1.md

```diff
@@ -0,0 +1,54 @@
+---
+files: n/a
+stepType: checks
+scripts: n/a
+week: 5
+step: 1
+name: Week 5 Step 1
+---
+Week 5 Step 1 ⬤◯◯◯ | 🕐 Estimated completion: 30 minutes
+
+## Laying out the groundwork
+
+### ✅  Task:
+
+- [ ]  Create a new [figma](figma.com) file named "Serverless Camp Project"
+- [ ]  In this file, create a flowchart showing the flow of information and components involved in your project including any Azure Functions and APIs you plan on using
+- [ ]  Export the flowchart as a png file 
+- [ ]  Create a branch named `week5`
+- [ ]  Create a folder in this repo where you will be saving all of your project related files, called "project"
+- [ ]  Upload the flowchart into the project folder and commit your changes to move on
+
+### Mapping it out
+
+To get started with building, students will first need to create a flowchart showing the flow of information and components they will include to make there project work. Nothing has to be finalized, but planning it out allows students to organize their ideas and thought processes.
+
+### Purpose of Programming Flowcharts
+
+- Plan before the development phase helps you save a lot of hours of work
+- Document your project and communicate how it works to others (this is really valuable when someone else is helping you debug your code)
+- Prevent major technical issues with your code before you event start
+- Ensure that you are more effectively using all the technologies available to you
+- You will need to refer back to the flowchart while developing to review your work and sometimes come up with new ideas
+
+<details>
+<summary>:exclamation: How do I create a flowchart?</summary>
+</br>
+Part A: Purpose
+1. Create a checklist of all of the components in your project you want to use, including APIs, functions, databases, packages/libraries, etc.
+3. Below each shape, create bullet points on what purpose they serve in your project (be as specific as you want to)
+
+Part B: Interactions
+1. Create a new circular shape/block for each of the components and write their names inside each shape
+2. Draw lines on all of the technologies that interact with each other
+3. Near each line, use the previous bullet points you wrote to describe the relationship between the technologies starting with action verbs (e.g. between JS code/front-end --> Azure Function "Sends an HTTP Post Request with uploaded image)
+
+Part C: Review
+1. Get a friend/mentor to look over the flowchart and try to explain back the program to you
```

Comment:
This should be in the checkbox ( maybe another step

Reply:
Did you mean checkbox for 1. 2. 3. or for each Part?

Reply:
each part sohuld be another step

Reply:
you need to create week 5 step 2, 3